app/views/info/_lead_metrics.html.erb

```diff
@@ -1,16 +1,33 @@
 <section id="lead-metrics">
   <% if lead_metrics %>
+    <%- if format == 'guide' %>
```

Comment:
I did it this way to future-proof it for smart answers, I think we're going
to need different wording in the view for them.

They're different because they don't have unique page views for each URL,
due to the way GA records its metrics, but they do have problem reports.
On 24 Jan 2015 21:42, "Jake Benilov" <notifications@github.com> wrote:

> In app/views/info/_lead_metrics.html.erb
> <https://github.com/alphagov/info-frontend/pull/38#discussion_r23499635>:
>
> > @@ -1,16 +1,33 @@
> >  <section id="lead-metrics">
> >    <% if lead_metrics %>
> > +    <%- if format == 'guide' %>
>
> I think the confusion that I went through could be avoided by having a
> boolean is_multipart flag instead of a format variable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/info-frontend/pull/38/files#r23499635>.
>